hack/run-ci-e2e-test.sh

```diff
@@ -103,10 +103,10 @@ fi
 # -flag x is allowed for non-boolean flags only(https://golang.org/pkg/flag/)
 
 # Test that the operator is running when the private key secret is not present
-OSDK_WMCO_test $OSDK "-run=TestWMCO/operator_deployed_without_private_key_secret -v -node-count=$NODE_COUNT -skip-node-deletion -ssh-key-pair=$KEY_PAIR_NAME --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION"
+OSDK_WMCO_test $OSDK "-run=TestWMCO/operator_deployed_without_private_key_secret -v -node-count=$NODE_COUNT -ssh-key-pair=$KEY_PAIR_NAME --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION"
```

Comment:
If this option is not getting used anywhere, shouldn't we remove the option from the test code also?

Reply:
As shown in README [here](https://github.com/openshift/windows-machine-config-operator#running-e2e-tests-on-a-cluster),
it can be used to avoid running of deletion test suite if we are not running the tests separately. However that is not the case, so I will go ahead and delete the code

Reply:
This implies there is a use case for it as folks might be using it in their development workflow. Please don't remove it then.

Reply:
Although, removing it on this change makes sense. The `-skip-node-deletion` can be kept as it is in the code...Removing the flag altogether might need a greater discussion...

Reply:
Yes. However, I am confused even when we use the flag as mentioned here `hack/run-ci-e2e-test.sh -s -k "openshift-dev" -n 2 `,  we still go ahead and run `destroy` test suite.